lib/app/routes/static.rb

```diff
@@ -17,6 +17,18 @@ class Static < Core
         erb :"site/about", locals: {active_languages: active_languages, upcoming_languages: upcoming_languages, planned_languages: planned_languages}
       end
 
+      get '/submitter-workflow' do
+         erb :"site/submitter-workflow",locals: { active_languages: active_languages }
```

Comment:
Line is too long. [86/80]<br>Use 2 (not 3) spaces for indentation.<br>Space missing after comma.